connect/http-client/src/main/java/org/camunda/connect/httpclient/impl/HttpConnectorLogger.java

```diff
@@ -59,4 +59,9 @@ public void ignoreConfig(String field, Object value) {
     logInfo("009", "Ignoring request configuration option with name '{}' and value '{}'", field, value);
   }
 
+  public ConnectorRequestException unableToExecuteRequest(int statusCode , String connectorResponse) {
+	    return new ConnectorRequestException(exceptionMessage("010", "Unable to execute HTTP request with Status Code : " 
```

Comment:
🔧 I'd rename this to `httpRequestError` or something similar and the message could also say something along `HTTP request failed with status code: 500...`
🔧 Let's also use the message template with parameters instead of string concatenation, like: `"HTTP request failed with Status Code: {} , Response: {}", statusCode, connectorResponse)`